packages/wonder-blocks-icon/src/components/phosphor-icon.tsx

```diff
@@ -52,6 +52,14 @@ type Props = Pick<AriaProps, "aria-hidden" | "aria-label" | "role"> & {
      * - `string`: an import referencing an arbitrary SVG file.
      */
     icon: PhosphorIconAsset | string;
+    /**
+     * Optional sizing unit for width and height: pixels or REMs.
+     *
+     * Defaults to px for backwards compatibility.
+     *
+     * A REM unit will scale with the user's font-size, so it is preferred.
+     */
+    sizingUnit?: "px" | "rem";
```

Comment:
Are there any cases where an icon should be in `px` instead of `rem`? If not, I think all icons can be in `rem` and we won't need this prop! That can be handled separately though since switching to rems may require more testing for existing use cases !

Reply:
I'm not sure. I mostly didn't want this to snowball into a larger change since the original scope of work was related to font theming, not icon sizing. If it didn't look so bad with font scaling inside of Links I would have tackled it separately!